src/libYARP_dev/src/devices/AnalogWrapper/AnalogWrapper.cpp

```diff
@@ -140,10 +140,10 @@ AnalogPortEntry &AnalogPortEntry::operator =(const AnalogPortEntry &alt)
 
  // closing anonimous namespace
 
-/**
-  * It reads the data from an analog sensor and sends them on one or more ports.
-  * It creates one rpc port and its related handler for every output port.
-  */
+/*
+ * It reads the data from an analog sensor and sends them on one or more ports.
+ * It creates one rpc port and its related handler for every output port.
+ */
```

Comment:
This comment block seems to be just a duplication of the doxygen comments in https://github.com/Nicogene/yarp/blob/cd3b2ec0c8742eb8a8e92117d9de81848179babb/src/libYARP_dev/src/devices/AnalogWrapper/AnalogWrapper.h#L67 , I think it can be safely removed.